packages/uxpin-merge-cli/src/steps/serialization/component/implementation/ComponentPropertyDefinition.ts

```diff
@@ -64,6 +64,7 @@ export interface PropertyTypeStructureMap {
   custom:{};
   date:{};
   element:{};
+  enum:{ label:string | number, value:string | number | ts.PseudoBigInt };
```

Comment:
what do you think about moving type to new interface EnumTypeStructure like e.g. UnionTypeStructure ?

Reply:
fixed; tbh i got inspired by lieral type below